docs/homepage.md

```diff
@@ -0,0 +1,146 @@
+page using a similar structure. None of the below information is needed for normal use or maintenance of the site.
+
+### Collections
+
+The homepage takes advantage of the `EntryCollection` model. (You will not have to make any changes to the 
+`EntryCollection` model, but if you want to see how it is constructed, it is found in /writings/models.py) This is 
+already connected to the admin, so the articles included in the collection can be selected and re-ordered there. 
+
+If you are creating a new collection, that will be done in the admin. From the admin homepage, click "Writings", 
+then "Collections", then "Add entry collection". Provide a name (what website admins will see as the name of the 
+collection) and a slug (usually the name, but all lowercase and using '-' instead of spaces. Make sure there are no
+spaces or special characters in the slug). Finally, add a reference to the collection in /newsilkroad/settings.py.
+You'll see a parameter called `CONSTANCE_CONFIG`. Add a new entry, following the pattern of the others:
+
+```
+'MY_NEW_COLLECTION': (
+    'slug-for-my-collection',
+    'Description of the collection', str
+),
+```
+
+### FeaturedEntryMixin
+In order to make sure you have access to the functions which make the articles in the collection available to the 
+template, you will need to subclass the `FeaturedEntryMixin`. You will also need to specify which `EntryCollection` will
+be used to generate the content for this site. That's where you'll reference the collections entry in `CONSTANCE_CONFIG` 
+in /newsilkroad/settings.
+
+The final result will look something like this:
+
+```
+from django.views.generic import TemplateView
+from writings.views import FeaturedEntryMixin
+
+class MyNewPage(FeaturedEntryMixin, TemplateView):
+    template_name = "website/my_template.html"
+    featured_config_key = 'MY_NEW_COLLECTION'
+```
+
+That will point the `FeaturedEntryMixin` to the correct collection, and include the entries in the context.
+
+### Use in Template
+
+The resulting entries will be available in the template as `featured_entry_set`.
+
+Because `EntryCollection` references `OrderedEntry` objects rather than `Entry` objects directly, attributes will be 
+accessed with what may seem to be an extra `.entry`. (See examples below.) If the extra `.entry` is omitted, nothing 
+will appear in that place.
+
+When referencing the content or description of the entry, you will want to use `content_rendered` or 
+`description_rendered`, respectively, as those have been parsed to be html-ready. You will also want to use the Django
+template function `safe` to ensure that Django does not duplicate the rendering that has already been done. The end
+result will look something like `{{ entry.entry.content_rendered | safe }}`
+
+There are two options for using the resulting context data in the template: iterating or retrieving individual entries.
+
+#### Iterating
```

Comment:
Yes you are right. Then that should be noted. Anything published in the collection will be shown which is different behavior from the  base `ConfiguredCollectionMixin`.